plugins/logitech-rallysystem/logitech-rallysystem.quirk

```diff
@@ -0,0 +1,11 @@
+# Logitech Rally System Audio, resides in TVHub
+[HIDRAW\VEN_046D&DEV_0885]
+Plugin = logitech_rallysystem
+GType = FuLogitechRallySystemAudioDevice
+Flags = ~updatable
```

Comment:
do you need the Flags line? I'm confused why it would be marked as updatable in the first place.